File: PooCC01CNetz/Semana2/Java/Ejercicios2/src/Estadistica.java
public class Estadistica {
    public static void main(String[] args) {
        Estadistica e = new Estadistica();
        //Declarar un arreglo
        double []arreglo = {1.2, 2.4, 3.6, 4.8};
        double med = e.media(arreglo);
        System.out.println("La media es: " + med);
    }

    public double media(double[] vec) {
        double med, acum = 0;
        for (int index = 0; index < vec.length; index++) {
            acum = acum + vec[index];            
        }
        med = acum / vec.length;
        return med;
    }
}

File: PooCC01CNetz/Semana2/Python/Ejercicios2/Estadistica.py
class Estadistica:
    def main(self):
        e = Estadistica()
        arreglo = [1.2, 2.4, 3.6, 4.8]
        med = e.media(arreglo)
        print("La media es:", med)

    def media(self, vec):
        acum = 0
        for value in vec:
            acum += value
        med = acum / len(vec)
        return med

if __name__ == "__main__":
    est = Estadistica()
    est.main()
